src/encoded/static/components/browse/BrowseView.js

```diff
@@ -259,38 +189,62 @@ class ResultTableContainer extends React.PureComponent {
 
     render() {
         const {
-            context, href, countExternalSets, session, browseBaseState, schemas, windowHeight,
-            selectedFiles, sortBy, sortColumn, sortReverse, windowWidth, isFullscreen, facets
+
+            // From Redux store or App.js:
+            context, schemas, currentAction, windowWidth, windowHeight, registerWindowOnScrollHandler, session,
+
+            // 4DN-Specific from Redux Store or App.js:
+            browseBaseState, isFullscreen, toggleFullScreen,
+
+            // From BrowseView higher-order-component (extends facets, removes type facet, etc)
+            facets, topLeftChildren, countExternalSets,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['topLeftChildren' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4660599)